rest-client/src/main/java/org/sagebionetworks/bridge/rest/RestUtils.java

```diff
@@ -486,15 +487,26 @@ public static UserSessionInfo signInWithSynapse(AuthenticationApi authApi, SignI
                 .synapseSignIn(SYNAPSE_LOGIN_URL, body).execute().body();
         String sessionToken = object.get("sessionToken").getAsString();
 
-        body = makeRequestBody(
+        JsonObject idClaim = new JsonObject();
+        idClaim.addProperty("userId", (String)null);
+        JsonObject claims = new JsonObject();
+        claims.add("id_token", idClaim);
+
+        body = makeRequestBody(claims, 
                 "clientId", "100018", 
                 "scope", "openid", 
                 "responseType", "code",
-                "claims", "{\"id_token\":{\"userid\":null}}",
                 "redirectUri", OAUTH_CALLBACK_URL);
         
-        object = retrofit.create(OauthConsent.class)
-                .oauthConsent(SYNAPSE_OAUTH_CONSENT, body, sessionToken).execute().body();
+        Response<JsonObject> response = retrofit.create(OauthConsent.class)
+                .oauthConsent(SYNAPSE_OAUTH_CONSENT, body, sessionToken).execute();
+        if (!response.isSuccessful()) {
+            System.out.println(response.code());
+            System.out.println(response.message());
+            System.out.println(response.body().toString());
+            System.out.println(response.errorBody().toString());
+        }
```

Comment:
We should remove these print statements.

Reply:
Yikes, removed (this was looking at the OAuth issue)